noxfile.py

```diff
@@ -70,52 +84,59 @@ def tests(session: nox.Session) -> None:
         )
 
     install(session, "-e", "./vectors")
-    if session.name == "tests-rust-debug":
-        install(
-            session,
-            "--config-settings=build-args=--profile=dev",
-            f".[{extras}]",
+
+    install_args = []
+    if build_args:
+        install_args.append(
+            f"--config-settings=build-args={' '.join(build_args)}"
         )
-    else:
-        install(session, f".[{extras}]")
+    install_args.append(f".[{extras}]")
+    install(session, *install_args)
 
     if session.venv_backend == "uv":
         session.run("uv", "pip", "list")
     else:
         session.run("pip", "list")
 
-    if session.name != "tests-nocoverage":
-        cov_args = [
+    cov_args = (
+        [
             "--cov=cryptography",
             "--cov=tests",
             "--cov-context=test",
         ]
-    else:
-        cov_args = []
-
-    if session.posargs:
-        tests = session.posargs
-    else:
-        tests = ["tests/"]
+        if coverage
+        else []
+    )
 
+    test_paths = session.posargs or ["tests/"]
     session.run(
         "pytest",
         "-n",
         "auto",
         "--dist=worksteal",
         *cov_args,
         "--durations=10",
-        *tests,
+        *test_paths,
     )
 
-    if session.name != "tests-nocoverage":
+    if coverage:
         [rust_so] = glob.glob(
             f"{session.virtualenv.location}/lib/**/cryptography/hazmat/bindings/_rust.*",
             recursive=True,
         )
         process_rust_coverage(session, [rust_so], prof_location)
 
 
+@nox.session(name="tests-abi3-py311")
```

Comment:
Is there a reason this didn't work as another `if` statement in teh same function?

Reply:
I can do it that way instead.  Pushed 69e697641